### ISSUE TITLE
Improve photon to prisma client error

### ISSUE THREAD

**Issue by matthewmueller** [prisma/prisma-client-js#444]
Right now we show this:

![image](https://user-images.githubusercontent.com/170299/73186369-e16c8a80-411f-11ea-8ff9-3e69d791c8e5.png)

It's great that we detect this, but it's not very actionable. I propose we reword this to the following:

---

Oops! Photon has been renamed to Prisma Client. Please make the following adjustments:

1. Rename `provider = "photonjs" ` to `provider = "prisma-client-js"` in your `schema.prisma` file.
2. Update your `package.json`'s `@prisma/photon` dependency to `@prisma/client`
3. Adjust `import { Photon } from '@prisma/photon'` to `import { PrismaClient } from '@prisma/client'` in your code.
4. Run `prisma2 generate` again

**Comment by timsuchanek** [comment]
Thanks a lot for reporting 🙏
This issue is fixed in the latest alpha version of `prisma2`.
You can try it out with `npm i -g prisma2@alpha`.

In case it’s not fixed for you - please let us know and we’ll reopen this issue!

**Comment by matthewmueller** [comment]
Thank you TimBot!
